uefi-raw/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 - Added `Boolean` type
 - Added `protocol::network::pxe` module.
 - Added conversions between `MacAddress` and the `[u8; 6]` type that's more commonly used to represent MAC addresses.
-
+- Added `DiskInfoProtocol`.
```

Comment:
nit: keep two blank lines after this (each version in the changelog is separated by two blank lines)